src/libraries/Common/src/System/Collections/Generic/BitHelper.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace System.Collections.Generic
 {
-    internal readonly ref struct BitHelper
```

Comment:
Looks like this change was made accidentally?

Reply:
@eiriktsarpalis no I was just reverting my previous change per https://github.com/dotnet/runtime/pull/78613#discussion_r1027464402